doc/contributing.md

```diff
@@ -420,7 +553,7 @@ Don't forget to commit the baseline image as well!
 The images should be pushed up into a remote repository using `dvc` (instead of
 `git`) as will be explained in the next section.
 
-#### Using data version control ([dvc](https://dvc.org)) to manage test images
+#### Using Data Version Control ([dvc](https://dvc.org)) to Manage Test Images
```

Comment:
Originally posted in https://github.com/GenericMappingTools/pygmt/pull/1335#pullrequestreview-685712272.

The level-4 title does not look good (Same for **Using mpl_image_compare** abvoe and **Using check_figures_equal** below):
![dvc](https://user-images.githubusercontent.com/50591376/123505228-7c102680-d690-11eb-99c7-d24bb16da863.png)

I check the file, only **Pull Request Workflow** section also has Level-4 titles: **General Guidelines for Making a Pull Request (PR)** and **General Process for Pull Request Review**. But they mainly contain some lists so they look okay.

Reply:
I added bold formatting to the section headers in https://github.com/GenericMappingTools/pygmt/pull/1339/commits/e7357aa64fa3c551ddc8230480e6d2522490309b. The only other option that can think of is converting the file to ReST, but since that would involve many formatting updates I would prefer that to be a separate issue where we could add good first issue and help wanted labels.